plasma_framework/contracts/src/exits/models/OutputGuardModel.sol

```diff
@@ -3,6 +3,7 @@ pragma experimental ABIEncoderV2;
 
 library OutputGuardModel {
     /**
+     * @dev The data structure that is being used for IOutputGuardHandler. Contains essential data related to output guard.
```

Comment:
Based on the documentation provided, I have hard time grasping what the OutputGuard is used for. Could we provide more information and references on what this is used? In general this useful for other data structures as well.